web/packages/teleport/src/Notifications/Notifications.tsx

```diff
@@ -46,6 +55,9 @@ const logger = Logger.create('Notifications');
 export function Notifications({ iconSize = 24 }: { iconSize?: number }) {
   const ctx = useTeleport();
   const { clusterId } = useStickyClusterId();
+  useStore(ctx.storeNotifications);
```

Comment:
we subscribed to the store here, this means any update to the store notifications should update the notifications but i'm not sure this is being done

here are the scenarios where the store can get updated:
- when you go to the access list page, the store refreshes
- when you review or delete an access list, the store updates (removes them from the store)

during my testing, i do not see the reviewed or deleted being removed from notifications unless you refresh the whole page.

Reply:
To get this working I had to change the way the logic for prepending the local notifications works, have another look and let me know what you think